extension-manifest-v3/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "10.0.7",
   "type": "module",
   "scripts": {
+    "postinstall": "npm run download-engines && npm run download-dnr-lists",
```

Comment:
don't we need `npm run licenses` here as well?

Reply:
Pointless, the `dist` is completely removed for each build, so licenses are generated during the build.